src/main/java/net/syscon/elite/web/handler/ControllerAdvice.java

```diff
@@ -262,4 +267,50 @@
                         .developerMessage(e.getMessage())
                         .build());
     }
+
+    @ExceptionHandler(JpaSystemException.class)
```

Comment:
This is currently handling the cell capacity error thrown from old Nomis, but that business rule will be duplicated in another PR.  However it seems like a good idea to catch any similar errors thrown out of Nomis and return a nice message - but each one we find should be duplicated in this app.